File: app/src/main/java/fr/socket/florian/dhome/view/MainFragment.java
package fr.socket.florian.dhome.view;

import android.support.annotation.StringRes;
import android.support.v4.app.Fragment;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;

public class MainFragment extends Fragment {
    protected void setTitle(@StringRes int title) {
        AppCompatActivity activity = (AppCompatActivity) getActivity();
        if(activity != null) {
            ActionBar actionBar = activity.getSupportActionBar();
            if(actionBar != null) {
                actionBar.setTitle(title);
            }
        }
    }
}

File: app/src/main/java/fr/socket/florian/dhome/view/MainFragment.kt
package fr.socket.florian.dhome.view

import androidx.annotation.StringRes
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment

abstract class MainFragment : Fragment() {
    protected fun setTitle(@StringRes title: Int) {
        (activity as AppCompatActivity?)?.supportActionBar?.setTitle(title)
    }
}
,